recipes/r-shinycustomloader/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set version = '0.9.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: shinycustomloader
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/shinycustomloader_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/shinycustomloader/shinycustomloader_{{ version }}.tar.gz
+  sha256: cee1f758eb30bc4cb76c3ac463482b31e2167150312e689c7ce0199177389dce
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-glue
+    - r-shiny
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-glue
+    - r-shiny
+
+test:
+  commands:
+    - $R -e "library('shinycustomloader')"           # [not win]
+    - "\"%R%\" -e \"library('shinycustomloader')\""  # [win]
+
+about:
+  home: https://github.com/emitanaka/shinycustomloader
+  license: GPL-2.0-or-later
```

License: GPL (= 3) is GPL-3.0-only
```suggestion
  license: GPL-3.0-only
```